server/src/main/kotlin/com/mobilejazz/colloc/util/StringUtils.kt

```diff
@@ -0,0 +1,23 @@
+package com.mobilejazz.colloc.util
+
+import java.net.URL
+
+fun String.toFullImportedFileURL() = URL("https://docs.google.com/spreadsheets/d/${this}/export?format=csv")
+
+object Android {
+
+  private val questionMarkAtStartRegex = "^(\\?)".toRegex()
+  private val referenceAtStartRegex = "^(@)".toRegex()
+  private val tagOpenRegex = "^(<)".toRegex()
+  private val tagCloseRegex = "^(>)".toRegex()
+  internal fun String.encodeLiterals(): String =
+    replace("%@", "%s")
+      .replace(referenceAtStartRegex, "\\\\@")
+      .replace(questionMarkAtStartRegex, "\\\\?")
+      .replace("'", "\\'")
+      .replace("&", "&amp;")
+      .replace(tagOpenRegex, "&lt;")
+      .replace(tagCloseRegex, "&gt;")
+      .replace("\"", "\\\"")
+}
```

Comment:
It could be just a `encodeAndroidLiterals()` where all the `private val` properties are within the `String.encodeLiterals()` function, so you don't need the `Android` object for nothing. 

Everything will be more compact and clear. 

**Note:** Rename file to `StringExtensions` instead of `Utils`

Reply:
you can keep it outside as a `lazy val` and set it as private so it's only used in this file but the Object is not needed.